examples/model_compress/amc/amc_search.py

```diff
@@ -28,27 +29,29 @@ def parse_args():
     parser.add_argument('--train_episode', default=800, type=int, help='number of training episode')
     parser.add_argument('--n_gpu', default=1, type=int, help='number of gpu to use')
     parser.add_argument('--n_worker', default=16, type=int, help='number of data loader worker')
-    parser.add_argument('--job', default='train_export', type=str, choices=['train_export', 'export_only'],
-                        help='search best pruning policy and export or just export model with searched policy')
-    parser.add_argument('--export_path', default=None, type=str, help='path for exporting models')
-    parser.add_argument('--searched_model_path', default=None, type=str, help='path for searched best wrapped model')
+    parser.add_argument('--suffix', default=None, type=str, help='path for searched best wrapped model')
```

Comment:
this path is for saving or loading the wrapped model?

Reply:
yes, the `searched_model_path` was used to save/load the temporary wrapped module before export, it is not necessary any more after changing to nni speedup.

Reply:
this is a path, but the name is "suffix". it is a little misleading

Reply:
Thanks,  the help string was wrong, fixed.